src/_guides/language/common-prob.md

```diff
@@ -0,0 +1,473 @@
+
+<div class="fails-sa" markdown="1">
+{% prettify dart %}
+void main() {
+  var map = {
+    'a': 7,
+    'b': 11,
+    'c': 13
+  }; // <= inferred to be Map<String, int>
+
+  [[highlight]]map['d'] = 1.5;[[/highlight]]  // 1.5 is not int!
+}
+{% endprettify %}
+</div>
+
+This can be fixed by explicitly defining the map's type to be
+`<String,dynamic>`.
+
+<div class="passes-sa" markdown="1">
+{% prettify dart %}
+void main() {
+  var map = [[highlight]]<String,dynamic>[[/highlight]]{
+    'a': 7,
+    'b': 11,
+    'c': 13
+  };
+
+  map['d'] = 1.5;
+}
+{% endprettify %}
+</div>
+
+Alternatively, if you only want this map to accept integers and floats,
+you can specify the type as `<String,num>`.
+
+<hr>
+
+<a name="constructor-initialization-list"></a>
+### Constructor initialization list super() call
+
+**Error:**  <code>super call must be last in an initializer list
+   (see https://goo.gl/EY6hDP): 'super(style)'.</code>
+
+This error occurs when the `super()` call is not last in a constructor's
+initialization list.
+
+**Fix:** Put the `super()` call last.
+
+DDC generates simpler code if it relies on the `super()` call appearing last.
+The following example generates an error in strong mode Dart:
+
+<div class="fails-sa" markdown="1">
+{% prettify dart %}
+HoneyBadger(Eats food, String name)
+  : _name = name,
+    [[highlight]]super[[/highlight]](food) { ... }
+{% endprettify %}
+</div>
+
+Fix the error by moving the `super()` call:
+
+<div class="passes-sa" markdown="1">
+{% prettify dart %}
+HoneyBadger(Eats food, String name)
+  : [[highlight]]super[[/highlight]](food),
+    _name = name { ... }
+{% endprettify %}
+</div>
+
+For more information, see [DO place the super() call last in a
+constructor initialization list](/guides/language/effective-dart/usage#do-place-the-super-call-last-in-a-constructor-initialization-list)
+in [Effective Dart](/guides/language/effective-dart/).
+
+<hr>
+
+{% comment %}
+### Invalid field override
+
+Strong mode currently disallows overriding a field with another field or with a getter/setter. Generally speaking making the superclass field into a getter/setter pair is a simple solution.  Note that we do allow an interface which declares a field to be implemented with a field, or with a getter/setter. Overriding a field with another field or with a getter/setter pair generally leaves the old field around using up memory in the object. It’s technically still possible to access the field, but most code simply leaves it around unused.
+
+We will eventually support field overrides in strong mode, but DDC does not support them yet.  For the time being, you can use the @virtual annotation from package:meta to make a field virtual (allowing it to be overridden), but the resulting code will not yet work in DDC.
+{% endcomment %}
+
+{% comment %}
+## Known issues
+Do we have any known issues or bugs to list here?
+{% endcomment %}
+
+<a name="other-resources"></a>
+## Other resources
```

Comment:
I might be inclined to move this info up to the top, rather than having a section dedicated to referring you to an identically named section in another page.

That could also free you up so you could make the Contents section just a couple of lists/paragraphs. (Very optional, but I'd be tempted.)